test/option.js

```diff
@@ -155,6 +166,18 @@ describe('option', () => {
             const parser = option({isDeprecated: true});
             assert.doesNotThrow(() => parser({}, LAZY_CONFIG), MissingOptionError);
         });
+
+        it('should not log warning on deprecated option', () => {
```

Comment:
"...on deprecated option with default value"

Reply:
It does not warn on deprecated options with default values already.
It does, however, on deprecated option without one.
If option is removed, it doesn't need the default value, and developer might not provide it, assuming, `{isDeprecated: true}` will not log any warnings. Currently, his assumptions are falsy.

Reply:
Renamed test title to "... without any value"